src/test/scripts/python/api/ping.py

```diff
@@ -8,13 +8,13 @@
 
 
 # ----------------------------------------------------------------------
-def ping(dateTime, pilotBeaconId, power, chronoId):
+def ping(instant, pilotBeaconId, power, chronoId):
   "This adds a Ping"
-  data = '{"dateTime":"' + formatDatetime(dateTime) + '", "power":"' + str(power) + '"}'
+  data = '{"instant":"' + formatDatetime(instant) + '", "power":"' + str(power) + '"}'
```

Comment:
You could have avoided a lot of pain by avoiding this name change!

Reply:
Yes but instant is more meaningful in this context